.github/workflows/rpm-build-and-test.yml

```
@@ -0,0 +1,104 @@
+name: RPM Build and Test
+
+env:
+  # TODO: Surely this is a GH variable of some sort for the name of the repo
+  NAME: argobots
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr-repos:
+        description: 'Any PR-repos that you want included in this build'
+        required: false
+      commit-message:
+        description: 'Commit message to use rather than the one from git'
+        required: false
+      rpm-test-version:
+        description: 'RPM version to test'
+        required: false
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: rpm-build-and-test-${{ github.head_ref  || github.run_id }}
```

I think you can use this variable for the workflow
```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

---

Ah, yes.  Another nice one I will push if I don't feel the squeeze of time closing in on me.

---

Agree